airflow/task/task_runner/standard_task_runner.py

```diff
@@ -53,6 +55,11 @@ def start(self):
         else:
             self.process = self._start_by_exec()
 
+        if self.process:
```

Comment:
Do we want to add an option to enable/disable this feature? It's adding a new thread to every task execution which I could see some users not loving.

Reply:
We could, I am not a big fan of creating new configuration/flag because it adds complexity but I also understand that some users do not want/need that. Maybe others have some thought?

Reply:
I think we should just go with always enabled, until and unless there are major concerns from community members/users here. If concerns are raised later, we can always add a flag at that time.